src/pages/AdminDashboard/AdminInvestment.tsx

```
@@ -1,9 +1,176 @@
+      <TabsBody 
+        animate={{ mount: { opacity: 1 }, unmount: { opacity: 0 } }}
+        placeholder=""
+        onPointerEnterCapture={() => {}}
+        onPointerLeaveCapture={() => {}}
+      >
+
+{activeTab === "investment1" && (
+  <div className="mt-10">
+
+  <Table data={data} tableHead={tableHead}>
+              {data.map((table: any, index: number) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)